components/CurrentBuildingInformation.js

```diff
@@ -91,24 +95,61 @@ function CurrentLocation () {
                 { latitude: coord.lb.coordinates[3].latitude, longitude: coord.lb.coordinates[3].longitude },
                 { latitude: coord.lb.coordinates[4].latitude, longitude: coord.lb.coordinates[4].longitude }])) {
 
-                setcurrentBuilding(coord.lb.name);
+                setCurrentBuilding(coord.lb.name);
 
             }
         }, 1000);
         return () => clearInterval(intervalId);
     });
 
-    currentFloor = (altitude - baseAltitude) / floorHeight;
-
-    if (currentFloor > 0) {
-        roundedCurrentFloor = "This is the " + (Math.round(currentFloor * 100) / 100).toFixed(0) + "rd floor";
-    }
-
-    if (currentBuilding === "") {
-        return ("");
-    }
-    return (currentBuilding + " - " + roundedCurrentFloor);
-
+    //Compute the floor level
+    // currentFloor = (altitude - baseAltitude) / floorHeight;
+    // roundedCurrentFloor = "Floor level: " + (Math.round(currentFloor * 100) / 100).toFixed(0);
+
+
+    return (
+        <TouchableOpacity style ={styles.layout}>
+        <View style ={{top: "2%"}}>
```

Comment:
Style in stylesheet, not inline (for consistency)

Reply:
Okay